src/lib.rs

```diff
@@ -697,6 +697,12 @@ impl<'ctx> Bindings<'ctx> {
             try!(writer.write("\n".as_bytes()));
         }
 
+        // TODO:
+
+        try!(writer.write("use objc;\n\n".as_bytes()));
```

Comment:
Instead of printing this here we should use something like what we do to insert the `__BindgenUnionField`, etc. See `CodegenResult::saw_union` and similar methods.